sdk/servicebus/service-bus/samples/typescript/src/receiveMessagesStreaming.ts

```diff
@@ -30,26 +36,49 @@ export async function main() {
   // - See session.ts for how to receive using sessions.
   const receiver = sbClient.createReceiver(queueName);
 
-  const processMessage = async (brokeredMessage) => {
-    console.log(`Received message: ${brokeredMessage.body}`);
-    await brokeredMessage.complete();
-  };
-  const processError = async (args) => {
-    console.log(`Error from error source ${args.errorSource} occurred: `, args.error);
-  };
-
   try {
-    receiver.subscribe(
+    const subscription = receiver.subscribe(
```

Comment:
Same set of changes need to be done in the corresponding JS file as well

Reply:
Yes, was waiting until the review for this sample was done to avoid having to keep doing it twice. :)

Reply:
This is done.